framework/tlvf/src/include/tlvf/CmduMessageTx.h

```diff
@@ -12,6 +12,8 @@
 #include <tlvf/CmduMessage.h>
 #include <tlvf/ieee_1905_1/tlvVendorSpecific.h>
 
+#define MTU_SIZE (size_t)1500
```

Comment:
Should be dynamically read from the interface

Reply:
This number is defined hardcoded on the transport. I think it is better to leave it.

Reply:
It's definitely not for this PR, but should probably be handled sometime in the future